frontend/tsconfig.json

```diff
@@ -25,12 +25,13 @@
     "paths": {
       "react": ["./node_modules/@types/react"]
     },
-    "incremental": true
+    "incremental": true,
   },
   "include": [
     "next-env.d.ts",
     "**/*.ts",
-    "**/*.tsx"
+    "**/*.tsx",
+    "**/*.js",
```

Comment:
I don't think you need the comma here. We would have the comma here in JS/TS, but this is a JSON file.